README.md

```diff
@@ -82,14 +94,11 @@ Import the styles into your project...
 
 ```scss
 @use 'node_modules/tbxforms/tbxforms.scss' with (
-    $tbxforms-error-colour: #f00,
     $tbxforms-text-colour: #000,
+    $tbxforms-error-colour: #f00,
 );
 ```
 
-Alternatively, variables can be defined in a centralised variables SCSS
-such as [tbxforms/static/sass/abstracts/\_variables.scss](https://github.com/torchbox/tbxforms/blob/main/tbxforms/static/sass/abstracts/_variables.scss).
```

Comment:
With this update, I couldn't get this to work 🤔 . I must be doing something wrong, though I wonder if it's something we should promote?

Reply:
This MR removes `_variables.scss` and looks like there are a few more files to add variables now. I'm also confused why we have `sass/govuk-overrides` when customisations are getting added in `sass/govuk` too.

Reply:
You can still override variables like this:

```
@use '../../../node_modules/tbxforms/dist/tbxforms.scss' with (
    $tbxforms-text-colour: $color--red,
);
```

I just cannot seem to get this to work anymore:

```
@use '../../../node_modules/tbxforms/dist/tbxforms.scss';

$tbxforms-text-colour: $color--red;
```

Reply:
OK, I guess that's fine then, I can see it's been also documented already.